spec/features/test_page_spec.rb

```diff
@@ -1,6 +1,6 @@
-feature 'test page' do
```

Comment:
I would remove these test pages once you know the code is ok, it isn't really relevant to the rest of the game now.